### PR TITLE
update manifests for kubeapps/manifest#21

### PR DESCRIPTION
**Pull request by prydonius**
see kubeapps/manifest#21

### CODE REVIEW

File: static/kubeapps-objs.yaml [hunk 1]
```diff
@@ -881,467 +818,515 @@ spec:
...
+        - mountPath: /monocular
+          name: cache
+        - mountPath: /monocular/config
+          name: config
+      - env:
+        - name: TILLER_NAMESPACE
+          value: kube-system
+        - name: TILLER_HISTORY_MAX
+          value: "0"
+        image: gcr.io/kubernetes-helm/tiller:v2.7.0
```
Comment (arapulido) [inline static/kubeapps-objs.yaml]: Shall we upgrade to 2.7.2? There is a security fix in 2.7.2: https://github.com/kubernetes/helm/releases/tag/v2.7.2

Comment (prydonius) [reply static/kubeapps-objs.yaml]: @arapulido I'll make the change in kubeapps/manifest#21 - can you review this there please?

**Comment by ngtuna** [comment]
Well there is a bunch of irrelevant changes to the original subject which was only "securing tiller". I know that you ran the script to synchronize the manifest from jsonnet format to YAML format. However that causes troubles to review changes. It looks like the sync script shuffled the whole manifest :P (or maybe github is too bad on showing `diff` of yaml format). So I think that'd be better to review the original PR on manifest (which contains clearer changes on tiller's stuff). When that PR is merged, we will just sync it again to YAML manifest, create a new PR (or update this one) and do nothing more than click merge button.

**Comment by ngtuna** [comment]
On a relevant note: have we decided to migrate the kubeapps/manifest to kubeapps/kubeapps repo and only work from there ? https://github.com/kubeapps/manifest/issues/19

**Comment by prydonius** [comment]
@ngtuna yeah sorry, my intention was for kubeapps/manifest#21 to be removed and this to just be merged after with the latest synced manifests.

r.e. kubeapps/manifest#19 - we're all in agreement we want to do this, but we haven't taken up the task due to higher priorities

**Pull request closed without merging**